fight.cpp

```diff
@@ -0,0 +1,41 @@
+#include "Fight.h"
+#include <iostream>
+Fight::Fight() {}
+Fight::Fight(int amount){
+    lenghtOfArray = amount;
+    arrayoffighters = new Fighter[amount];
```

Comment:
Harsh variable name. Please, rename from "**arrayoffighters**" to "**arrayOfFighters**".